griptape/drivers/file_manager/griptape_cloud_file_manager_driver.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+
+import logging
+import os
+import uuid
+from typing import Optional
+from urllib.parse import urljoin
+
+import requests
+from attrs import Attribute, Factory, define, field
+from azure.core.exceptions import ResourceNotFoundError
+from azure.storage.blob import BlobClient
```

Comment:
Are there any concerns with revealing that we're using azure storage?

Reply:
No, that was a decision in the GTC design to be OK with. GTC reserves the right to change the underlying storage, but that would be prefaced with a ~ 'storage_type' field on the asset so the driver can be updated to use the correct client (ex: S3)